Admin/ArticleAdmin.php

```diff
@@ -0,0 +1,266 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\ArticleBundle\Admin;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Sonata\AdminBundle\Admin\AbstractAdmin;
+use Sonata\AdminBundle\Admin\AdminInterface;
+use Sonata\AdminBundle\Datagrid\ListMapper;
+use Sonata\AdminBundle\Form\FormMapper;
+use Sonata\AdminBundle\Show\ShowMapper;
+use Sonata\ArticleBundle\Model\Article;
+use Sonata\ArticleBundle\Model\ArticleInterface;
+use Sonata\ArticleBundle\Model\Fragment;
+use Sonata\CoreBundle\Validator\ErrorElement;
+
+/**
+ * @author Florent Denis <florent.denis@ekino.com>
+ */
+class ArticleAdmin extends AbstractAdmin
+{
+    /**
+     * @var array
+     */
+    protected $datagridValues = array('_sort_by' => 'updatedAt', '_sort_order' => 'DESC');
+
+    /**
+     * @var string
+     */
+    protected $fragmentClass;
+
+    /**
+     * @param string $code
+     * @param string $class
+     * @param string $baseControllerName
+     * @param string $fragmentClass
+     */
+    public function __construct($code, $class, $baseControllerName, $fragmentClass)
+    {
+        parent::__construct($code, $class, $baseControllerName);
+
+        $this->fragmentClass = $fragmentClass;
+    }
+
+    /**
+     * Inspired from Twig_Extensions_Extension_Text.
+     *
+     * {@inheritdoc}
+     */
+    public function toString($object)
+    {
+        $value = $object->getTitle();
+        $length = 80;
+        $separator = '...';
```

Comment:
IMO this is not a separator but a suffix according to the usage.

BTW, this can now be handled by CSS, can not be?